pokemongo_bot/cell_workers/evolve_all_worker.py

```diff
@@ -52,36 +52,40 @@ def work(self):
                 self._release_evolved(release_cand_list_ids)
 
     def _should_run(self):
-        # Will skip evolving if user wants to use an egg and there is none
-        skip_evolves = False
-
-        if self.config.evolve_all:
-            return skip_evolves
+        # Skip Evolving if there is no config
+        if not self.config.evolve_all or self.config.evolve_all is "NONE":
+            return False
 
         # Pop lucky egg before evolving to maximize xp gain
-        use_lucky_egg = self.config.use_lucky_egg
+        use_lucky_egg = True if self.config.use_lucky_egg is True else False
```

Comment:
I thought "what if they won't enter a boolean in the config file" but I guess that would be their fault.

Fixed it.